servers/dsub/jobs/controllers/utils/time_convert.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+from dateutil.tz import tzlocal
+
+TIME_FRAME_MAP = {
+    'HOURS_1': datetime.timedelta(minutes=60),
+    'HOURS_8': datetime.timedelta(minutes=60 * 8),
+    'HOURS_24': datetime.timedelta(days=1),
+    'DAYS_7': datetime.timedelta(days=7),
+    'DAYS_30': datetime.timedelta(days=30)
+}
+
+
+def time_frame_to_start_time(time_frame):
+    """
+    Map a time frame to start time.
+
+    Args:
+        time_frame_to_start_time (time_frame): 'HOURS_1', 'HOURS_8', 'HOURS_24',
```

Comment:
The arg is named time_frame. The type is str

Reply:
Fixed.